package.json

```diff
@@ -52,9 +57,13 @@
     "react-native": "0.68.2",
     "react-native-app-auth": "^6.4.3",
     "react-native-device-info": "^10.3.0",
+    "react-native-draggable-flatlist": "^4.0.1",
```

Comment:
It doesn't seem like adding the deps above this line to `devDependencies` is right.  If, according to our [peer dependencies strategy](https://github.com/lifeomic/react-native-sdk/blob/master/CONTRIBUTING.md#peer-dependencies-strategy), you think they shouldn't be peer dependencies - I think they need to go into `dependencies`.  Otherwise these won't be included in a way the npm-published lib can use them.

Reply:
Yes, fixed.